src/main/java/com/jaoafa/mymaid4/command/Cmd_Wire.java

```diff
@@ -0,0 +1,188 @@
+ */
+
+
+package com.jaoafa.mymaid4.command;
+
+import cloud.commandframework.ArgumentDescription;
+import cloud.commandframework.Command;
+import cloud.commandframework.context.CommandContext;
+import cloud.commandframework.meta.CommandMeta;
+import cloud.commandframework.types.tuples.Triplet;
+import com.jaoafa.mymaid4.lib.CommandPremise;
+import com.jaoafa.mymaid4.lib.MyMaidCommand;
+import com.jaoafa.mymaid4.lib.MyMaidLibrary;
+import com.sk89q.worldedit.IncompleteRegionException;
+import com.sk89q.worldedit.bukkit.WorldEditPlugin;
+import com.sk89q.worldedit.math.BlockVector3;
+import com.sk89q.worldedit.regions.CuboidRegion;
+import com.sk89q.worldedit.world.World;
+import io.leangen.geantyref.TypeToken;
+import org.bukkit.Location;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Bat;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Player;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+import org.bukkit.util.Vector;
+
+import java.util.Collections;
+
+import static com.jaoafa.mymaid4.Main.getWorldEdit;
+
+public class Cmd_Wire extends MyMaidLibrary implements CommandPremise {
+    @Override
+    public MyMaidCommand.Detail details() {
+        return new MyMaidCommand.Detail(
+            "Wire",
+            Collections.singletonList("leadunit"),
+            "指定した2点間にリードを張ったり、撤去したりします。"
+        );
+    }
+
+    @Override
+    public MyMaidCommand.Cmd register(Command.Builder<CommandSender> builder) {
+        return new MyMaidCommand.Cmd(
+
+            builder
+                .meta(CommandMeta.DESCRIPTION, "指定した2点間にリードを張ります。座標を指定しない場合、WorldEditで選択した2点間にリードを張ります。")
+                .literal("set", "add", "connect")
+                .argumentTriplet(
+                    "pos1",
+                    TypeToken.get(Vector.class),
+                    Triplet.of("x1", "y1", "z1"),
+                    Triplet.of(Integer.class, Integer.class, Integer.class), // バニラのようにintで入ってきた場合は+0.5blockし、doubleの場合はそのまま代入するようにしたい
+                    (sender, triplet) -> new Vector(triplet.getFirst(), triplet.getSecond(), triplet.getThird()),
+                    ArgumentDescription.of("1つ目のx・y・z座標。リードを付けられている側。")
+                )
+                .argumentTriplet(
+                    "pos2",
+                    TypeToken.get(Vector.class),
+                    Triplet.of("x2", "y2", "z2"),
+                    Triplet.of(Integer.class, Integer.class, Integer.class), // バニラのようにintで入ってきた場合は+0.5blockし、doubleの場合はそのまま代入するようにしたい
+                    (sender, triplet) -> new Vector(triplet.getFirst(), triplet.getSecond(), triplet.getThird()),
+                    ArgumentDescription.of("2つ目のx・y・z座標。リードを持っている側。")
+                )
+                .handler(this::setWire)
+                .build(),
+
+            builder
+                .meta(CommandMeta.DESCRIPTION, "WorldEditで選択した2点間にリードを張ります。1座標目がリードを付けられている側、2座標目がリードを持っている側。")
```

Comment:
コマンドの説明が正しくないように思えます

```suggestion
                .meta(CommandMeta.DESCRIPTION, "WorldEditで選択した2点間にリードを張ります。")
```

Reply:
未修整では？

Reply:
ｧｱﾞ～～～:((: ◜◡‾:)):～～～？？！！

Reply:
かなしい